src/hubot-cah.coffee

```diff
@@ -155,4 +158,11 @@ module.exports = (robot) ->
   robot.hear /^cah skip$/i, (res) ->
     game.db.answers = []
     game.db.blackCard = game.deal_card('black')
-    res.send game.game_state_string()
\ No newline at end of file
+    res.send game.game_state_string()
+
+  robot.hearspond /cah decks$/i, (res) ->
+    res.reply "Available decks: " + deck.availableDecks().join(', ')
```

Comment:
Maybe just one command that uses _.difference or whatever the lodash command is, and prints Active and Inactive on 2 lines? This is fine, just thinking that might ultimately be more useful